### ISSUE TITLE
All new UI layout

### ISSUE THREAD

**Issue by MickL** [Aloshi/EmulationStation#801]
With the release of RasberryPi 4 with 4k, what do you guys think of an all new UI? I always felt like the design looks outdated, especially compared to Apple tvOS, Netflix, PS4, etc.

I would love to do some layouts but have limited time only. I got something super sleek and simple in my mind.

Maybe you could make an open request where designers all around the world could send in ui/ux designs and you or the community votes for the best one?

**Comment by daylerees** [comment]
Hi @MickL, I wonder if it would be worth starting with some UX wireframes. I work in product development, and normally we'll start by testing low-fidelity wireframes in front of people to gauge whether we've built something that feels good. Once we've confirmed this, we'll then add the paint on top.

It would be great to get more design and UX into open-source though! They certainly exist, but I do feel that there's a lesser representation!

(But I do share a desire to see a pretty - and highly usable - ES in the future. :) )

**Comment by MickL** [comment]
Yes thats definitely a good idea but sometimes its just nice for a designer to go for it. Especially when it comes to animations. After discovering look and feel its okay to go backwards and do some wireframes then. But ofcourse there is no right or wrong. 

Anyway as I heard RetroPie used a fork of this project. Also there is a new gamelauncher called Pegasus that might replsce ES someday. But they could need a nice UI too :D
